src/components/CardItem.js

```diff
@@ -1,24 +1,30 @@
 import React, { useState } from "react";
-import styled from "styled-components";
+import styled from "styled-components/macro";
+
+import Heart from "react-animated-heart";
 
 function CardItem(props) {
   const [readMore, setReadMore] = useState(false);
+  const [isClick, setClick] = useState(false);
```

Comment:
Please rename this state! It's not very precise and doesn't describe what it is for, better would be:
```suggestion
  const [isFavorite, setIsFavorite] = useState(false);
```